src/core/core.scale.js

```diff
@@ -512,7 +512,14 @@ class Scale extends Element {
 		let i, ilen, tick;
 		for (i = 0, ilen = ticks.length; i < ilen; i++) {
 			tick = ticks[i];
-			tick.label = helpers.callback(tickOpts.callback, [tick.value, i, ticks], me);
+			const ctx = {
```

Comment:
I don't really like the change of callback parameters, because it will require a lot of people to change code when upgrading. Could just pass the locale as new parameter?

Other than this, then changes look good. And this too, if not considering then amount of callbacks in the wild.

Reply:
done